test/JDBC/expected/BABEL-sp_helpdb-vu-verify.out

```diff
@@ -0,0 +1,60 @@
+sp_helpdb
+GO
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#int#!#nvarchar#!#nvarchar#!#tinyint
+master#!#<NULL>#!#jdbc_user#!#1#!#2022-10-10 #!#<NULL>#!#<NULL>
+tempdb#!#<NULL>#!#jdbc_user#!#2#!#2022-10-10 #!#<NULL>#!#<NULL>
+msdb#!#<NULL>#!#jdbc_user#!#4#!#2022-10-10 #!#<NULL>#!#<NULL>
+test_db1#!#<NULL>#!#jdbc_user#!#5#!#2022-10-10 #!#<NULL>#!#<NULL>
+~~END~~
+
+
+-- Executing sp_helpdb with already existing dbname as an input
+sp_helpdb 'master'
+GO
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#int#!#nvarchar#!#nvarchar#!#tinyint
+master#!#<NULL>#!#jdbc_user#!#1#!#2022-10-10 #!#<NULL>#!#<NULL>
```

Comment:
I think it won't always succeed because the created date will change. We can create a table and insert the procedure output into it, like we did in sp_helpsrvrolemember